include/z64message.h

```diff
@@ -249,12 +249,12 @@ typedef struct MessageContext {
     /* 0x12046 */ s16 blockSunsSong;
     /* 0x12048 */ u8 ocarinaButtonIndex;
     /* 0x1204A */ s16 ocarinaButtonsPosY[5];
-    /* 0x12054 */ s16 unk12054[6]; // First, second and third digits in lottery code guess
```

Comment:
~~The indexing for this array is weird with the undocumented index parameters, but are you saying the first three are the actual code and the three that follow are the guess value the player enters?~~
I should stop reviewing as I'm falling asleep, didn't realize you didn't write the comment.